src/api/public/apidocs/OBS-v2.9.yaml

```diff
@@ -0,0 +1,114 @@
+openapi: 3.0.0
+servers: []
+info:
+  description: Open Build Service API
```

Comment:
You can use the description field to give an intro to the API.
Some basic questions can be answered here, like what's the purpose of the API? When would I use it? Does it need authentication? Are there any conventions? Is it versioned, and if so, how do I specify the version?

You can even use markdown for this, for example:
https://github.com/SUSE/happy-customer/blob/master/glue/public/api_docs/swagger/swagger.yaml#L4